sdks/python/apache_beam/runners/portability/stager.py

```diff
@@ -62,6 +62,7 @@
 from urllib.parse import urlparse
 
 from packaging import version
+from pip._internal.operations import freeze
```

Comment:
let's not use internal APIs. a future upgrade to pip may break this  api call in previously released versions of Beam, this happened before: https://github.com/pypa/pip/issues/5243#issuecomment-381422449 .  Running `pip freeze` command line would be more reliable.

Quick search, but feel free to look more:

https://stackoverflow.com/questions/49923671/are-there-any-function-replacement-for-pip-get-installed-distributions-in-pip

Reply:
sounds good, did with cmd pip